### PR TITLE
Ghes workflow auth 15735

### LINKED ISSUE #15735: Fix GHES workflow for Authenticating with Desktop docs
**This issue is reserved for MLH fellow program**

## What article is affected

Inaccurate documentation of the GHES workflow for [authenticating with GitHub desktop](https://help.github.com/en/desktop/getting-started-with-github-desktop/authenticating-to-github-using-the-browser).

## What are the desired changes in the docs

To close this issue, you'll need to run through the process of connecting to a GitHub Enterprise Server instance with GitHub Desktop. When you pick this up, please let @mattpollard know—he'll provide access to an instance for you to test with!

You'll update the following to close this issue.

- "Authenticating to GitHub" article's "[Authenticating an account on GitHub Enterprise](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github#authenticating-an-account-on-github-enterprise)" section
  - Rename section to "Authenticating an account on GitHub Enterprise Server" (use variable for "GitHub Enterprise Server")
  - Update step 4
    - Refer to "an account on your GitHub Enterprise Server instance" (use variable for "your GitHub Enterprise Server instance")
    - Instruct reader to type the URL for their GitHub Enterprise Server instance (not their credentials 😅)
  - Add a step that describes what happen after clicking **Continue** in step 4
  - Remove references to two-factor authentication via SMS (which isn't possible with GitHub Enterprise Server)

### PR DESCRIPTION
**Pull request by cmwilson21**
### Why:

Closes [15735](https://github.com/github/docs/issues/15735)

### What's being changed:
Renamed section to "Authenticating an account on GitHub Enterprise Server" using variables.
Updated step 4 and added what happens after you click "continue" on step 4.
Added screen shot of Enterprise sign in form.
Removed reference to two factor auth. 

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by steveward — CHANGES_REQUESTED** [review]
Thanks so much for working on this @cmwilson21! I've gone through and added comments about screenshots and wording that needs to be updated since we've made a bunch of minor changes in the app. 😬  Apologies for throwing additional things on, but it'd be great to get all this updated. If you have any questions let me know!

File: content/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github.md [hunk 1]
```diff
@@ -23,25 +23,28 @@ Before you authenticate, {% data reusables.desktop.get-an-account %}
...
-## Authenticating an account on {% data variables.product.prodname_enterprise %}
+## Authenticating an account on {% data variables.product.prodname_ghe_server %}
 
 {% data reusables.user-settings.password-authentication-deprecation-desktop %}
 
 {% data reusables.desktop.mac-select-desktop-menu %}
 {% data reusables.desktop.mac-select-accounts %}
 {% data reusables.desktop.choose-product-authenticate %}
-4. To add a {% data variables.product.prodname_enterprise %} account, type your credentials under "Enterprise server address," then click **Continue**.
+4. To add an account on {% data variables.product.product_location_enterprise %}, type the URL for your instance under "Enterprise server address," then click **Continue**.
```
Comment (steveward) [inline content/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github.md:41]:
```suggestion
4. To add an account on {% data variables.product.product_location_enterprise %}, type the URL for your instance under "Enterprise address," then click **Continue**.
```

File: content/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github.md [hunk 2]
```diff
@@ -23,25 +23,28 @@ Before you authenticate, {% data reusables.desktop.get-an-account %}
 {% data reusables.desktop.mac-select-accounts %}
 3. To the right of "{% data variables.product.prodname_dotcom_the_website %}," click **Sign In**.
   ![The Sign In button for GitHub](/assets/images/help/desktop/mac-sign-in-github.png)
-4. In the "Sign in" pane, click **Sign in using your browser**. {% data variables.product.prodname_desktop %} will open your default browser.
-  ![The Sign in using your browser link](/assets/images/help/desktop/sign-in-browser.png)
+{% data reusables.desktop.sign-in-browser %}
 
   {% data reusables.user-settings.password-authentication-deprecation-desktop %}
```
[the remark this answers is not in the document]
Comment (cmwilson21) [reply content/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github.md:28]: Got it! Will remove this reusable and the others mentioned in the review and will update per suggestions. Thanks!

**Comment by cmwilson21** [comment]
@steveward Thanks again for leaving the suggestions and feedback! The screenshots have been updated now, except for the Windows ones. I'm on Mac but I have a Windows machine here I will use and download Desktop on to grab those screenshots. Those bits will be updated soon! I'll reach out when they are done.

**Comment by cmwilson21** [comment]
Hey @steveward :) I updated the two windows images to reflect the recent changes. They looked fine on my screen, but when I see the preview in my codespace, they look a bit blurry. Can you check them out? Let me know if anything else needs to be done on them. Thanks!

**Review by steveward — APPROVED** [review]
Changes look great to me, thanks @cmwilson21!

**Pull request merged**